.github/workflows/ci.yml

```diff
@@ -100,7 +100,7 @@ jobs:
         cache-suffix: ${{ matrix.nox_session }}-${{ matrix.python }}-${{ runner.os }}
 
     - name: Run the check
-      run: uv run --with nox --frozen nox -s '${{ matrix.nox_session }}'
+      run: uvx nox -s '${{ matrix.nox_session }}'
```

Comment:
I had added `--frozen` here originally to make sure that `uv.lock` was not changed. However, `uvx` runs `nox` from an isolated virtual environment independent of the local `uv.lock`.  Probably. 😅